app/views/users/new.html.haml

```diff
@@ -0,0 +1,22 @@
+%section.register.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %fieldset
+        = form_for @user, html: {role: "form", class: "form-horizontal"} do |f|
+          %header
+            %h1 Register
+          .form-group
+            = f.label :email, "Email Address", {class: "control-label col-sm-2"}
```

Comment:
Once you get that fixed, you can just do `=f.text-field :email` and it will spit out all of this for you.